emperor/support_files/js/view-controller.js

```diff
@@ -83,7 +83,7 @@ define([
     this.$header.css('margin', '0 auto');
     this.$header.css('width', '100%');
 
-    this.$body = $('<div name="emperor-view-controller-body"></div>');
```

Comment:
I believe this should still be `emperor-view-controller-body`, the gridDiv in turn should have a different name i.e. `emperor-grid-div` or something like that.